tests/data_extraction/test_mixture_metadata_extraction.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from lebedigital.raw_data_processing.metadata_extraction \
+    .mixture_metadata_extraction import extraction
+
+
+def test_extraction():
+    """Tesing the mixture metadata extraction on a single example"""
+
+    # setting up the test example
+    input = '../usecases/MinimumWorkingExample/Data/Mischungen/2014_08_05 Rezeptur_MIII.xlsx'
+    target_data = {'Cement -- Quantity in mix': 330, 'Cement -- Specific gravity': 3.123,
+                   'Cement -- Annotation': 'CEM I 42,5 R', 'Water (total) -- Quantity in mix':
+                    175, 'Water (total) -- Specific gravity': 1, 'Water (total) -- Annotation': 
+                    '---', 'Water (effective) -- Quantity in mix': 
+                    '---', 'Water (effective) -- Specific gravity': 
+                    '---', 'Water (effective) -- Annotation': '---', 'Air -- Quantity in mix': 
+                    '---', 'Air -- Specific gravity': '---', 'Air -- Annotation': '---', 
```

Comment:
There seem to be values missing.
Where is "Cement -- Quantity in mix" or "Water (total) -- Specific gravity"?